README.md

```diff
@@ -37,6 +37,7 @@ definitions are stored. Supported file extensions are `.zs` and `.zserio`.
 - The ternary operator only works when used in functions.
 - Offsets and indexed offsets are not implemented yet.
 - Constraints are not implemented yet.
+- The `bazel` build environment does not work on Windows.
```

Comment:
What are the errors? Should we create an issue for this?

Reply:
Good idea. Let me open an issue for it.

The issues I have found so far:
- I couldn't get `buildifier` to run under Windows. It fails with:
```
FATAL: ExecuteProgram(C:\users\...\...(...\execroot\__main__\bazel-out\x64_windows-fastbuild\bin\buildifier.bash) failed:
ERROR: src/main/native/windows/process.cc(202): CreateProcessW("C:\users\...\...(...\execroot\__main__\bazel-out\x64_windows-fastbuild\bin\buildifier.bash"): %1 is not a valid Win32 application.
```
- [internal/parser/BUILD.bazel](https://github.com/woven-planet/go-zserio/blob/main/internal/parser/BUILD.bazel) uses `sh_binary`, which doesn't work on Windows.

Reply:
Added issue https://github.com/woven-planet/go-zserio/issues/128 .

Reply:
I also updated the description in the readme file to be more specific.